src/utils/sources-tree/getURL.js

```diff
@@ -54,13 +55,14 @@ function _getURL(source: Source, defaultDomain: string): ParsedURL {
         group: `${protocol}//${host || ""}`
       };
 
-    case "webpack:":
     case "ng:":
+    case "webpack:":
       return {
         ...def,
         path: pathname,
         filename,
-        group: `${protocol}//`
+        group: `${protocol}//`,
```

Comment:
I'm surprised that this doesn't include the hostname. How that we've got WHATWG URL, I'd have expected more of the logic in this function to collapse into 
```
group: `${protocol}//${host || ""}`
```